packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1934,8 +1923,8 @@ export class MergeTree {
 	}
 
 	public obliterateRange(
-		start: SequencePlace,
-		end: SequencePlace,
+		start: number | InteriorSequencePlace,
```

Comment:
i know @titrindl didn't like number, but that is all we support in the case that is checked in. once we support sides i think this should change from number to InteriorSequencePlace, i don't think it should ever be both